lib/routemaster/middleware/metrics.rb

```diff
@@ -22,6 +22,9 @@ def call(request_env)
           rescue Routemaster::Errors::BaseError => e
             increment_response_count(response_tags(e.env))
             raise e
+          rescue Faraday::TimeoutError => e
```

Comment:
(If the circuit is open we won't report any metrics)

Reply:
Yep, but we would know how many timeouts we received before the breaker was tripped. We anyway don't push any metric once a circuit trips. We would have to record circuitbox metrics for that information.